sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -96,6 +96,17 @@ public override string ContentType
             }
         }
 
+        /// <summary>
+        ///   Hidden property that translates the <see cref="MessageWithMetadata.ContentType"/> property to/from a string.
+        /// </summary>
+        ///
```

Comment:
```suggestion
        /// <summary>
        ///   This member is intended to allow the string-based <see cref="ContentType" /> in this class to be
        ///    translated to/from the <see cref=Azure.Core.ContentType" /> type used by the <see cref="MessageWithMetadata" />
        ///    base class.
        /// </summary>
        ///
```

I'm not advocating for this specific text, just some additional context around the reason we need this is because of the base class member having a different type.   That's not clear without digging into the class hierarchy.